dialoguekit/utils/dialogue_reader.py

```diff
@@ -86,7 +90,7 @@ def json_to_dialogues(
 
     dialogues = []
     for dialogue_data in data:
-        dialogue = Dialogue(agent_id, user_id)
+        dialogue = Dialogue(agent_id, user_id, conversation_id)
```

Comment:
`conversation_id` should be retrieved from `dialogue_data` before the Dialogue instantiation, something like: `dialogue_data.get(_FIELD_CONV_ID, None)`.

Reply:
Done